kibana.md

```diff
@@ -889,6 +889,12 @@ readonlyrest_kbn.kibana_custom_css_inject_file: '/tmp/custom.css'
 readonlyrest_kbn.kibana_custom_js_inject: '$(".global-nav__logo").hide(); alert("hello!")'
 ```
 
+Alternatively, it's possible to load the JS from a file in the filesystem:
+
+```
```

Comment:
Worth mentioning for this new property (and its CSS counterpart) that if you use relative paths, you end up pointing to kibana home, i.e. 
```
readonlyrest_kbn.kibana_custom_js_inject: 'config/custom.js'
```
Will refer to `$KBN_HOME/config/custom.js` which is the same directory where `kibana.yml` can normally be found.

Reply:
updated ✔️